tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -33,97 +33,106 @@ public function testFix($expected, $input = null)
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     * @param null|string $expectedWithKeepMultipleSpacesAfterComma
+     *
+     * @dataProvider testFixProvider
+     */
+    public function testFixKeepMultipleSpacesAfterComma($expected, $input = null, $expectedWithKeepMultipleSpacesAfterComma = null)
```

Comment:
So instead of
```diff
     'test method call with tab' => array(
         '<?php xyz($a=10, $b=20, $c=30);',
         "<?php xyz(\$a=10 , \$b=20 ,\t \$c=30);",
+        "<?php xyz(\$a=10, \$b=20,\t \$c=30);",
     ),
```
you want:
```diff
     'test method call with tab' => array(
         '<?php xyz($a=10, $b=20, $c=30);',
         "<?php xyz(\$a=10 , \$b=20 ,\t \$c=30);",
     ),
+    'test method call with tab' => array(
+        '<?php xyz($a=10, $b=20, $c=30);',
+        "<?php xyz(\$a=10, \$b=20,\t \$c=30);",
+        array('keep_multiple_spaces_after_comma' => false),
+    ),
```
?

If yes, I'd rather moving everything in another test/provider